test/unit/dfc_test.py

```diff
@@ -117,59 +123,61 @@ def test_union_of_two(self):
         dfc2.store("second", second)
 
         final_dfc = dfc1.union(dfc2)
-        self.assertEqual(["first"], dfc1.get_table_names())
-        self.assertEqual(["second"], dfc2.get_table_names())
-        self.assertEqual(["first", "second"], final_dfc.get_table_names())
+        assert dfc1.get_table_names() == ["first"]
+        assert dfc2.get_table_names() == ["second"]
+        assert final_dfc.get_table_names() == ["first", "second"]
```

Comment:
Is this particular ordering of names guaranteed? Would `["second", "first"]` indicate failure? Could convert both sides to sets.

Reply:
Yup, `get_table_names` uses `sort` so it guarantees order (with the intention to make the results stable for easy unit testing).